### DISCUSSION TITLE
Assisting new members on hub

### DISCUSSION THREAD

**Discussion by FloriSpoelstra** [alkem-io/alkemio#396] [Ideas] [answered]
I think we  have to assist first-timers on the platform a bit more so they find their way around faster.

One of the issues I see is that people are not always finding their way to the apply buttons (we have far more sign-ups than we have applications). From a user perspective I think it would be great if you can apply directly for an ecoverses/challenge/opportunity as part of the sign-up proces. Or we need to remind people somehow that they still have to apply for something to become part of a community. 

Apart from that it would be great if we can introduce people somehow to the functions of the platform.

Originally posted by @FloriSpoelstra as part of discussion [#382](/alkem-io/alkemio/discussions/382)

**Comment by techsmyth** [comment, marked as answer]
Good to look at how this would work, as the communities are by design hierarchical - and while this may need to change in the future it would be a huge one to pick up now. 

So assuming the communities are hierarchical, the best option might be the following (a) the user applies to the challenge (b) this generates two approvals on the server: one for admission to the ecoverse and one for admission to the challenge. The latter would fail until the first one is approved, with an error message to this effect shown to the challenge admin. 

This is needed as otherwise the challenge admin role would in effect be able to add users to the ecoverse community - which I do not think is what we want.

Thoughts?

**Reply by FloriSpoelstra** [reply]
If it is difficult to implement I wouldn't bother for now. As a first step it would really help if people get a welcome e-mail once they signed up to the platform, with in that e-mail a reminder to apply for challenges/communities they want to stay informed of. Let's see if the problem still exist after that.

**Reply by techsmyth** [reply]
Hi Flori, 

Since a couple of weeks, when users sign up they do get an email to confirm their email address - but that is primarily for email verification. As part of the notifications epic we could look to also trigger welcome emails. Will note that into that epic.

Related is that each user is also shown a welcome screen encouraging them to explore after they have signed up more:

![image](https://user-images.githubusercontent.com/30729240/130483402-b1b9b6b5-c11b-4cfd-967a-5a9866a1b722.png)
We can easily update that content. Please let me know if you think other text should go in there. 

Good to chat this through more...

Cheers, Neil
